app/workers/send_user_invitation_worker.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class SendUserInvitationWorker < ApplicationJob
+  queue_as :default
+  include Net
```

Comment:
This should not be needed.

Reply:
But it is 😳

Reply:
Why? What happens when you remove this line? I bet `test/workers/send_user_invitation_worker_test.rb` passes entirely 😉

Reply:
It throws something like _SendUserInvitationWorker::Net::SMTPError is not defined_.